libraries/rush-lib/src/cli/scriptActions/PhasedScriptAction.ts

```diff
@@ -258,6 +260,8 @@ export class PhasedScriptAction extends BaseScriptAction<IPhasedCommandConfig> {
           'Normally a phased command expects "rush install" to have been manually run first. If this flag is specified, ' +
           'Rush will automatically perform an install before processing the current command.'
       });
+
+      this._variantParameter = this.defineStringParameter(VARIANT_PARAMETER);
```

Comment:
We still need this parameter if `this._alwaysInstall === true`, as well. Only time we don't is when it is `undefined`.

Reply:
Handled in https://github.com/microsoft/rushstack/pull/4944.